v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/iowrapper/JdbcIoWrapper.java

```diff
@@ -258,6 +258,10 @@ private static ImmutableList<TableConfig> autoInferTableConfigs(
     ImmutableList<String> discoveredTables =
         schemaDiscovery.discoverTables(dataSource, config.sourceSchemaReference());
     ImmutableList<String> tables = getTablesToMigrate(config.tables(), discoveredTables);
+    if (tables.isEmpty()) {
```

Comment:
Could we cover this change in `JdbcIoWrapperTest.java`?

Reply:
added